clients/venice-client/src/main/java/com/linkedin/venice/fastclient/RetriableAvroGenericStoreClient.java

```diff
@@ -163,18 +168,20 @@ public void streamingBatchGet(
       Set<K> keys,
       StreamingCallback<K, V> callback) throws VeniceClientException {
     if (!longTailRetryEnabledForBatchGet) {
+      // if this class is used only for retry of singleGet, no need to retry here as this is batchGet
```

Comment:
confusing comment. how about - if longtailretry is not enabled , simply return the original future.

Reply:
Changed thanks.